Core/Core/Offline/OfflineModeInteractor.swift

```diff
@@ -22,6 +22,7 @@ public protocol OfflineModeInteractor {
     func isOfflineModeEnabled() -> Bool
     func observeIsOfflineMode() -> AnyPublisher<Bool, Never>
     func observeNetworkStatus() -> AnyPublisher<NetworkAvailabilityStatus, Never>
+    func filePath(sessionID: String, fileID: String, fileName: String) -> String
```

Comment:
I'd keep this interactor lean and wouldn't include screen specific logic here. If we start adding functions that are needed for the different screens, we could end up with pretty large class with multiple responsibilities.

Reply:
This is a common interactor for all screens handling offline files, I wanted to maintain the path in one place. Up for suggestion for a better placement though!

Reply:
Refactored it into a separate interactor.